app/src/main/java/mozilla/lockbox/support/Constant.kt

```diff
@@ -26,6 +26,14 @@ object Constant {
         val syncTimeout: Long = 20
     }
 
+    /*
+     *  Used to turn features in the app on and off, where true denotes that the feature is available when
+     *  the app is run and false denotes that the feature is not available when the app is run.
+     */
+    object FeatureToggle {
+        const val delete = false
+    }
```

Comment:
I think I'd break `FeatureToggle` out into its own file. I'd also use longer names like just `delete`.

We're going to put all the logic to decide if we're going to expose a feature in this class, and then just use that class where we implement this feature.

Reply:
We'll want to use calculated properties looking at `BuildConfig` properties here, and most likely setting something up in `build.gradle` to generate build flavors.